### ISSUE TITLE
`Sync` namespace

### ISSUE THREAD

**Issue by iherman** [w3c/sync-media-pub#36]
[3§](https://w3c.github.io/sync-media-pub/sync-media.html#syncmedia) defines a new namespace for the features defined in this specification. Does this also means that _all_ other terms are considered to be part of the `smil` namespace (e.g., `http://www.w3.org/ns/SMIL`)? If so, it is worth emphasizing this, as well as formally defining a `sync` namespace URL.

**Comment by marisademeglio** [comment]
Yes, that's the intention -- SMIL3 + `sync` extensions (and minus large chunks of SMIL3). 

Do you think we could get a namespace from W3C, something more "official" than `w3c.github.io/sync-media-pub` ?

**Comment by iherman** [comment]
> 
> Do you think we could get a namespace from W3C, something more "official" than `w3c.github.io/sync-media-pub` ?

Eventually, yes. While we are in the incubation phase it does not really matter (there should be a Note in the spec saying that the final URL will come) and we get into the position of publishing a CG report then I will try to get one.
